### DISCUSSION TITLE
part-ii/2021-101/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#358] [Question comments]
Part II 2021 Number Fields: Paper 2, Section II, 20G 

https://questions.tripos.org/part-ii/2021-101/

**Comment by artie2000** [comment]
<details>
<summary>First two paragraphs:</summary>

Write the minimal polynomial as $m_\alpha(X)=X\cdot g(X)+k$, and set $\beta=g(\alpha)$.

The $r/m$ form follows by unpacking the definitions.

</details>

<details>
<summary>Third paragraph:</summary>

$R$ is a $\mathbb{Z}$-submodule of $\mathcal{O}_K$, and so free and finitely generated (by the structure theorem, say). Clearing denominators, a $\mathbb{Q}$-basis for $K$ gives a $\mathbb{Q}$-basis for $K$ contained in $R$, so $R$ has rank $[K:\mathbb{Q}]$.

The rest follows from the same bookwork for $\mathcal{O}_K$.

</details>
<details>
<summary>Last part:</summary>

Take $I=m\mathcal{O}_K$ and $J=R$, so that $IJ=mR$.

Then $[R:I]=[R:IJ]=[R:I][m\mathcal{O}_K:mR]$, so $m\mathcal{O}_K=mR$, so $\mathcal{O}_K=R$.

</details>

**Reply by callumhobbis** [reply]
How do you get the $r/m$ form with $r\in R$? I can do it if $R=\mathcal O_K$ via the minimum polynomial of $n\alpha$, but otherwise I don't know.

**Reply by callumhobbis** [reply]
Never mind, I figured it out. 

<details>
<summary>Hint / where I went wrong in my thinking</summary>

Don't fall into the trap of adapting the proof of "$\forall \alpha\in K$ algebraic $\exists n\in\mathbb Z^+$ such that $n\alpha\in\mathcal O_K$" like I did - we literally just use the previous parts of the question.

</details>
<details>
<summary>My solution for the r/m part</summary>

For any $\alpha\in K$, we may write $\alpha=r/s$ for $r,s\in R$. But note since $s\in R$ and $1\in R$, $\mathbb Z[s]\subset R$ and so $\exists s'\in R$ with $ss'=k\in\mathbb Z\setminus\{0\}$. Then $\alpha=r/(k/s')=rs'/k$.

</details>

**Reply by FTcode** [reply]
Since we're multiplying ideals _of_ $R$, isn't $R$ the identity? Then we'd have $m\mathcal O_k R =m\mathcal O_k$, not $mR$?
